packages/shared/styles/components/molecules/SfRadio.scss

```diff
@@ -1,77 +1,85 @@
-@import '../../variables';
-:root {
-  --radio-font-family: var(--body-font-family-secondary);
-  --radio-font-size: var(--font-size-small);
-  --radio-font-weight: var(--body-font-weight-primary);
-  --radio-line-height: 1.6;
-  --radio-transition: background-color .25s cubic-bezier(1, .5, .8, 1);
-  --radio--is-active-background: var(--c-light);
-  --radio__label-font-size: var(--font-size-extra-big);
-  --radio__label-font-weight: 500;
-  --radio__checkmark-transition: border .25s cubic-bezier(1, .5, .8, 1);
-  --radio__checkmark-size: 1.4375rem;
-  --radio__checkmark-border-color: var(--c-gray-variant);
-  --radio--is-active__checkmark-border-color: var(--c-primary);
-  @media screen and (min-width: $desktop-min) {
-    --radio-font-size: var(--font-size-small);
-    --radio__label-font-size: var(--font-size-big);
-  }
-}
+@import "../../variables";
+
 .sf-radio {
-  $this: &;
-  display: flex;
-  font-family: var(--radio-font-family);
-  font-size: var(--radio-font-size);
-  font-weight: var(--radio-font-weight);
-  line-height: var(--radio-line-height);
-  transition: var(--radio-transition);
+  display: var(--radio-display, flex);
+  font: var(
+    --radio-font,
+    font(
+      --radio-font,
+      var(--body-font-weight-primary),
+      var(--font-size-small),
+      1.6,
+      var(--body-font-family-secondary)
+    )
+  );
+  transition: var(
+    --radio-transition,
+    background-color 0.25s cubic-bezier(1, 0.5, 0.8, 1)
+  );
+
   &.is-active {
-    background-color: var(--radio--is-active-background);
+    background: var(--radio-background, var(--c-light));
   }
-  &--transparent {
-    &.is-active {
-      background-color: transparent;
+
+  &__label {
+    display: var(--radio-label-display, flex);
+    font-size: var(--radio-label-font-size, var(--font-size-extra-big));
```

Comment:
@aniamusial use font(--radio-label-font, ...) here "19. use always font() when you want need even one font property"